### PR TITLE
added readme

### PR DESCRIPTION
**Pull request by c-kalenyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+- Maven, Docker
+- Lombok, Mapstruct, Swagger
+- Junit, Mockito, Testcontainers
+
+---
+
+### How to run:
+
+---
+
```
Comment (Elena-Bruyako) [inline README.md:32]: Add - clone repo 
Do I need to install Java Development Kit (JDK) ?

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comment

File: README.md (hunk 1 above)
Comment (c-kalenyk) [reply README.md:32]: About JDK, we're just running a docker image here which is isolated, is it not?

**Review by okuzan — APPROVED** [review]
better include emojis, versions for spring boot and MySQL, and loom video is recommended

**Pull request merged**